### PR TITLE
Admin: allow users to change the site icon.

### LINKED ISSUE #9095: Dashboard: provide a way to change the site icon
This card in the Jetpack dashboard shows an icon but it's not really clear where it came from or what it represents.

captura de pantalla 2018-03-20 a la s 13 50 38

It would be good if users could take action on this, and change it. The same control in Calypso takes users in Jetpack sites to the Customizer in those sites and Rick Banister points that we could do just that for this in p1521564928000107-slack-jetpack-design

captura de pantalla 2018-03-20 a la s 13 54 11

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #9095

#### Changes proposed in this Pull Request:

* add a button that takes the user to the Customizer. It opens in the Site Identity section so the user can change the site icon.

Tagging as Needs Design Review because the link to disconnect was previously below the site icon. With the addition of the button, we have now too much unnecessary blank space on the right.

### Without moving the link

Captura de Pantalla 2019-05-10 a la(s) 19 53 07

### This PR currently, with the link on the right

Captura de Pantalla 2019-05-10 a la(s) 20 02 39

#### Testing instructions:

* Go to the Jetpack admin
* click on the CHANGE button below the site icon
* ensure you're taken to Customizer > Site Identity
* add an icon
* close the Customizer
* ensure that upon coming back to the Jetpack admin, the new icon is shown

#### Proposed changelog entry for your changes:

* Admin: users can now go and change the site icon from the Jetpack admin

### CODE REVIEW

File: _inc/lib/admin-pages/class.jetpack-react-page.php [hunk 1]
```diff
@@ -387,6 +387,7 @@ function jetpack_current_user_data() {
 			'edit_posts'         => current_user_can( 'edit_posts' ),
 			'publish_posts'      => current_user_can( 'publish_posts' ),
 			'manage_options'     => current_user_can( 'manage_options' ),
+			'customize'          => current_user_can( 'customize' ),
```
Comment (jeherve) [inline _inc/lib/admin-pages/class.jetpack-react-page.php]: What's the difference between `manage_options` and `customize` in terms of permissions? Could we use `manage_options` here?

**Review by jeherve — COMMENTED** [review]
While it works, I think it's a bit confusing; when I see "change" here, I expect clicking the button will allow me to change something with my site's connection to WordPress.com:

image

I would not expect this link to lead me to the customizer at all. 

I'd love to have a design eye on this.

**Comment by scottsweb** [comment]
I find this a little confusing also. The 'change' option within the site connection group seems to imply that I am changing the connection in some way, not the icon. 

If this is the only place we can expose this option and it is important we do so, then I recommend it remains hidden until hover, a little like Twitter:

Screenshot 2019-05-13 at 12 23 41

We would then probably need to apply the same approach to the site icon and the Gravatar?

Screenshot 2019-05-13 at 12 24 51

Although I appreciate that is probably outside the scope of this PR :)

File: _inc/lib/admin-pages/class.jetpack-react-page.php (hunk 1 above)
Comment (eliorivero) [reply _inc/lib/admin-pages/class.jetpack-react-page.php]: That option is specifically to ensures the user has access to the Customizer.

**Comment by scottsweb** [comment]
@eliorivero I like that idea, I think a small edit icon could work well. I wonder if there is any prior art we can lean on as an example? I haven't seen something like this yet. 

If you want us to experiment with a few options then let me know.

**Comment by eliorivero** [comment]
Yes @scottsweb would be awesome if you could produce something. Thanks! 👍

**Pull request closed without merging**